Mathlib/Algebra/Polynomial/Roots.lean

```diff
@@ -434,6 +437,12 @@ theorem aroots_monomial [CommRing S] [IsDomain S] [Algebra T S]
     (monomial n a).aroots S = n • ({0} : Multiset S) := by
   rw [← C_mul_X_pow_eq_monomial, aroots_C_mul_X_pow ha]
 
+variable (R S) in
+theorem aroots_map (p : T[X]) [CommRing S] [Algebra T S] [Algebra S R] [Algebra T R]
```

Comment:
I guess this could be a simp lemma?
```suggestion
@[simp]
theorem aroots_map (p : T[X]) [CommRing S] [Algebra T S] [Algebra S R] [Algebra T R]
```